modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-rc-no-slip.i

```diff
@@ -0,0 +1,180 @@
+mu=1.1
+rho=1.1
+advected_interp_method='average'
+velocity_interp_method='rc'
+force_boundary_execution=true
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 10
+    ymin = -1
+    ymax = 1
+    nx = 100
+    ny = 20
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
+  fv_bcs_integrity_check = true
+[]
+
+[Variables]
+  [u]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
+    initial_condition = 1
+  []
+  [v]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
+    initial_condition = 1
+  []
+  [pressure]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
+  []
+[]
+
+[FVKernels]
+  [mass]
+    type = NSFVKernel
+    variable = pressure
+    advected_quantity = 1
+    advected_interp_method = ${advected_interp_method}
+    velocity_interp_method = ${velocity_interp_method}
+    vel = 'velocity'
+    pressure = pressure
+    u = u
+    v = v
+    mu = ${mu}
+    rho = ${rho}
+    ghost_layers = 2
+    force_boundary_execution = ${force_boundary_execution}
+  []
+
+  [u_advection]
+    type = NSFVKernel
```

Comment:
I've renamed this to `NSFVMomentumAdvection`. I also renamed a lot of other classes. The naming I believe is now consistent with what I've done for INSAD (e.g. our incompressible CFEM CG implementation). However, since I *believe* that I should be able to re-use all these `NSFV` objects for both incompressible and compressible simulations, I've kept the prefix as `NSFV` as opposed to something like `INSFV`.